runtime/include/vendor_version.h

```diff
@@ -31,4 +35,8 @@
  *
  * #define VENDOR_SHORT_NAME "ABC"
  * #define VENDOR_SHA "1a2b3c4"
+ *
+ * Example usage for inclusion of a vendor version string.
+ * This value will be set to the system property java.vm.version.
```

Comment:
I think this should say 'will be *used* to set the system property'.

Reply:
Or: This value will be stored in the system property java.vm.version.

Reply:
Changed to `This value will be stored to` to avoid confusion.